vspace/src/main/webapp/WEB-INF/views/staff/spaces/edit.html

```diff
@@ -1,29 +1,29 @@
-<html layout:decorate="~{layouts/main_staff}">
-
-<body>
-	<div class="main-content" layout:fragment="content">
-		<h1>Edit Space: [(${spaceForm.name})]</h1>
-
-		<form method="POST"
-			th:action='@{|/staff/space/${spaceId}/edit?${_csrf.parameterName}=${_csrf.token}|}' th:object="${spaceForm}"
-			enctype="multipart/form-data" style="margin-top: 40px">
-
-			<div class="form-group row">
-				<label for="name" class="col-md-2 col-form-label">Space Name:</label> 
-				<textarea type="text" class="form-control col-md-10" id="name" rows="1" cols="30"
-					 th:text="${spaceForm.name}" name="name"></textarea>
-			</div>
-
-			<div class="form-group row">
-				<label for="description" class="col-md-2 col-form-label">Description:</label>
-				<textarea class="form-control col-md-10" rows="5" cols="30"
-					id="description" name="description" th:text=${spaceForm.description} ></textarea>
-			</div>
-
-			<button class="btn primary-btn" type="submit" value="submit">Save Space</button>
-			<a th:href="@{|/staff/space/${spaceId}|}" class="btn btn-light">Cancel</a>
-		</form>
-	</div>
-</body>
-</html>
-
+<html layout:decorate="~{layouts/main_staff}">
+
+<body>
+	<div class="main-content" layout:fragment="content">
+		<h1>Edit Space: [(${spaceForm.name})]</h1>
+
+		<form method="POST"
+			th:action='@{|/staff/space/${spaceId}/edit?${_csrf.parameterName}=${_csrf.token}|}' th:object="${spaceForm}"
+			enctype="multipart/form-data" style="margin-top: 40px">
+
+			<div class="form-group row">
+				<label for="name" class="col-md-2 col-form-label">Space Name:</label> 
+				<textarea type="text" class="form-control col-md-10" id="name" rows="1" cols="30"
+					 th:text="${spaceForm.name}" name="name"></textarea>
+			</div>
+
+			<div class="form-group row">
+				<label for="description" class="col-md-2 col-form-label">Description:</label>
+				<textarea class="form-control col-md-10" rows="5" cols="30"
+					id="description" name="description" th:text=${spaceForm.description} ></textarea>
+			</div>
+
+			<button class="btn primary-btn" type="submit" value="submit">Save Space</button>
+			<a th:href="@{|/staff/space/${spaceId}|}" class="btn btn-light">Cancel</a>
+		</form>
+	</div>
+</body>
+</html>
+

```

Comment:
did you change anything in this file? if yes, what?

Reply:
file endings are wrong (I assume that's what changed)

Reply:
No, I didn't change anything. I got the updated code changes on merge of `VSPC-120`

Reply:
they are still wrong. please fix.

Reply:
nver mind